packages/formation/sass/modules/_m-nav-sidebar.scss

```diff
@@ -370,32 +370,3 @@ $level-3-hover-padding: 8px 12px 8px 30px;
   }
 
 }
-
-// This is the trigger-button on mobile
-.va-btn-sidebarnav-trigger {
```

Comment:
Why move this into `_m-buttons.scss`?

Without looking, my guess is that `_m-nav-sidebar.scss` is only included in `static-pages.entry.css`, and not in the application bundles, whereas `_m-button.scss` is included in both. If only the static pages use this styling, we probably shouldn't include it in the application bundles.

Mind you, once the design system team [culls the unused styles](https://github.com/department-of-veterans-affairs/va.gov-team/issues/17593#issuecomment-748370271) during the application and content builds, this will be less meaningful, but I don't know when that's going to happen yet.

At any rate, if I were to look for this `sidebarnav` class, I'd look in `_m-nav-sidebar.scss`. :man_shrugging:

Reply:
Good point. We already had some styling for `.va-btn-sidebarnav-trigger` in `_m-buttons.scss` so I flipped a coin and decided to consolidate things into that file. But perhaps I should move all that stuff into this file instead?